run/image-processing/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/GoogleCloudPlatform/golang-samples/run/image-processing
 
-go 1.12
+go 1.13
```

Comment:
Are we sure we want to bump this?

Reply:
Why wouldn't we? Updating to go 1.13 is one of the goals of this PR.

Reply:
Possibly updating to new versions is not something which should be routinely done?

Reply:
It's probably not a big deal. But, this signifies the Go code version compatibility. So, if we aren't using any 1.13+ features (I don't think so), then it seems like we should keep it at the lower value. If we don't need 1.13+ features, what is the benefit of bumping it?

I'm thinking about a user who isn't on 1.13 yet. They might see this and have some concern about copying the sample. It's fine to keep the Dockerfile on the latest version, but I am not sure about `go.mod` files.

In general, samples have supported the oldest version supported by GAE. But, that's clearly not relevant here.

Reply:
Reverted